Header header
float32 minimum_range
float32 maximum_range
